README.md

```diff
@@ -1,15 +1,17 @@
-# The Westfield Datetime Helper
+# Datetime Helper
 
-A collection of useful utilities for projects that have to deal with dates, times, and timezones.
+A collection of useful utilities for projects that have to deal with dates, times, and timezones, with a focus on rails projects that enforce the use of Zulu Time.
 
-### Current Status: Rspec Matchers done
+### Current Status: Feature Complete
 
 1. The `rspec` matcher `be_zulu_time` has been implemeted.
 2. The `zulu_time` `ActiveModel` validator has been implemented.
+2. `in_zulu_time` ActiveModel::Serializer helper method has been implemented.
 
```

Comment:
I would replace the current status section with 
```
### Features
 
1. `rspec` matcher `be_zulu_time` 
2. `zulu_time` `ActiveModel` validator 
3. `in_zulu_time` ActiveModel::Serializer helper
```

If i saw "Current Status: Feature Complete" I would read it as "Not open for PRs"